client/driver/qemu.go

```diff
@@ -117,12 +117,12 @@ func (d *QemuDriver) Fingerprint(cfg *config.Config, node *structs.Node) (bool,
 	matches := reQemuVersion.FindStringSubmatch(out)
 	if len(matches) != 2 {
 		delete(node.Attributes, qemuDriverAttr)
-		return false, fmt.Errorf("Unable to parse Qemu version string: %#v", matches)
+		if !currentlyEnabled {
```

Comment:
Can we just leave this the way it was? The client which invokes the fingerprinter is going to print the error.